index.html

```diff
@@ -27,7 +24,7 @@ <h1>Wild flowers</h1>
 
       <p> Here are photographs of honey-suckles, red columbine, tiger lilies and orhids that can be found throughout the Nlakapmux territory they are mainly bloom throughout the spring and early summer months.</p>
 
-  <a href="https://www.nntc.ca/docs/nntc_territory_map.pdf"
+      <a href="https://www.nntc.ca/docs/nntc_territory_map.pdf">NNTC Territory Map</a>
```

Comment:
This `a` tag was opened but never closed. I put some link text in there and closed it.